[2021/11/29 09:43:30]========== Start ==========
[2021/11/29 09:43:30][I][main] Python 3.9.9 (main, Nov 20 2021, 21:30:06) 
[GCC 11.1.0]
[2021/11/29 09:43:30][I][main] Auto_RRSIandWiFiSAR_Report.py 1.0.0.1
[2021/11/29 09:43:31][I][initPath] Delete Folder: ./input/WillyWJ_Chen
[2021/11/29 09:43:31][I][initPath] Create Folder: ./input/WillyWJ_Chen
[2021/11/29 09:43:31][I][initPath] Delete Folder: ./output/WillyWJ_Chen
[2021/11/29 09:43:31][I][initPath] Create Folder: ./output/WillyWJ_Chen
[2021/11/29 09:43:31][E][mapping] Unexpected Error: [Errno 2] No such file or directory: './Mapping/SanchezPeng/Mapping.json'
[2021/11/29 09:43:31][I][parseLog] ------- Start Parsing Log -------
[2021/11/29 09:43:31][I][parseLog] ------- Finish Parsing Log -------
[2021/11/29 09:43:31][I][log_to_excel] ------- Parsing Log to Excel -------
[2021/11/29 09:43:31][E][log_to_excel] Unexpected Error: list index out of range
[2021/11/29 09:43:31]========== End ==========
[2021/11/29 09:43:46]========== Start ==========
[2021/11/29 09:43:46][I][main] Python 3.9.9 (main, Nov 20 2021, 21:30:06) 
[GCC 11.1.0]
[2021/11/29 09:43:46][I][main] Auto_RRSIandWiFiSAR_Report.py 1.0.0.1
[2021/11/29 09:43:46][E][initPaht] Unexpected Error: [Errno 2] No such file or directory: '/home/sanchez/Desktop/RDTool/Automation_RSSI_WiFiSARQUERY/input/input_WillyWJ_Chen'
[2021/11/29 09:43:46][E][initPaht] Unexpected Error: [Errno 2] No such file or directory: '/home/sanchez/Desktop/RDTool/Automation_RSSI_WiFiSARQUERY/output/output_WillyWJ_Chen'
[2021/11/29 09:43:46][E][mapping] Unexpected Error: [Errno 2] No such file or directory: './Mapping/SanchezPeng/Mapping.json'
[2021/11/29 09:43:46][E][main] Unexpected Error: [Errno 2] No such file or directory: '/home/sanchez/Desktop/RDTool/Automation_RSSI_WiFiSARQUERY/input/input_WillyWJ_Chen'
[2021/11/29 09:43:46]========== End ==========
[2021/11/29 09:45:05]========== Start ==========
[2021/11/29 09:45:05][I][main] Python 3.9.9 (main, Nov 20 2021, 21:30:06) 
[GCC 11.1.0]
[2021/11/29 09:45:05][I][main] Auto_RRSIandWiFiSAR_Report.py 1.0.0.1
[2021/11/29 09:45:05][E][initPaht] Unexpected Error: [Errno 2] No such file or directory: '/home/sanchez/Desktop/RDTool/Automation_RSSI_WiFiSARQUERY/input/input_WillyWJ_Chen'
[2021/11/29 09:45:05][E][initPaht] Unexpected Error: [Errno 2] No such file or directory: '/home/sanchez/Desktop/RDTool/Automation_RSSI_WiFiSARQUERY/output/output_WillyWJ_Chen'
[2021/11/29 09:45:05][E][pullData] Unexpected error: [Errno 13] Permission denied: '/home/sanchez'
[2021/11/29 09:45:05][E][main] Unexpected Error: [Errno 2] No such file or directory: '/home/sanchez/Desktop/RDTool/Automation_RSSI_WiFiSARQUERY/input/input_WillyWJ_Chen'
[2021/11/29 09:45:05]========== End ==========
[2021/11/29 09:58:03]========== Start ==========
[2021/11/29 09:58:03][I][main] Python 3.9.9 (main, Nov 20 2021, 21:30:06) 
[GCC 11.1.0]
[2021/11/29 09:58:03][I][main] Auto_RRSIandWiFiSAR_Report.py 1.0.0.1
[2021/11/29 09:58:03][E][initPaht] Unexpected Error: [Errno 2] No such file or directory: '/home/sanchez/Desktop/RDTool/Automation_RSSI_WiFiSARQUERY/input/input_WillyWJ_Chen'
[2021/11/29 09:58:03][E][initPaht] Unexpected Error: [Errno 2] No such file or directory: '/home/sanchez/Desktop/RDTool/Automation_RSSI_WiFiSARQUERY/output/output_WillyWJ_Chen'
[2021/11/29 09:58:03][E][main] Unexpected Error: [Errno 2] No such file or directory: '/home/sanchez/Desktop/RDTool/Automation_RSSI_WiFiSARQUERY/input/input_WillyWJ_Chen'
[2021/11/29 09:58:03]========== End ==========
[2021/11/29 09:59:10]========== Start ==========
[2021/11/29 09:59:10][I][main] Python 3.9.9 (main, Nov 20 2021, 21:30:06) 
[GCC 11.1.0]
[2021/11/29 09:59:10][I][main] Auto_RRSIandWiFiSAR_Report.py 1.0.0.1
[2021/11/29 09:59:10][E][initPaht] Unexpected Error: [Errno 2] No such file or directory: '/home/sanchez/Desktop/RDTool/Automation_RSSI_WiFiSARQUERY/input/input_WillyWJ_Chen'
[2021/11/29 09:59:10][E][initPaht] Unexpected Error: [Errno 2] No such file or directory: '/home/sanchez/Desktop/RDTool/Automation_RSSI_WiFiSARQUERY/output/output_WillyWJ_Chen'
[2021/11/29 09:59:10][E][main] Unexpected Error: [Errno 2] No such file or directory: '/home/sanchez/Desktop/RDTool/Automation_RSSI_WiFiSARQUERY/input/input_WillyWJ_Chen'
[2021/11/29 09:59:10]========== End ==========
